module-cellular/test/unittest_URC.cpp

```diff
@@ -279,78 +279,6 @@ TEST_CASE("+Qind: FOTA")
     }
 }
 
-TEST_CASE("+Cusd")
```

Comment:
maybe it's worth adding some test cases for the new implementation? especially that you're using regex :P

Reply:
Yeah, I'm already working on them...

Reply:
Decided to temporarily withdraw the UTs, see https://appnroll.atlassian.net/browse/MOS-812?focusedCommentId=69817.